format/Layout.md

```diff
@@ -89,8 +89,9 @@ maximum of 2^31 - 1 elements. We choose a signed int32 for a couple reasons:
 
 Any relative type can be nullable or non-nullable.
 
-Nullable arrays have a contiguous memory buffer, known as the validity (or
-null) bitmap, whose length is large enough to have 1 bit for each array slot.
+Nullable arrays have a contiguous memory buffer, known as the null (or
+validity) bitmap, whose length is large enough to have 1 bit for each array
+slot.
```

Comment:
I agree. There's also the SIMD question — if these buffers are word-aligned then there won't be concerns (? someone with more expertise should opine) with aligned allocations